dotCMS/src/main/java/com/dotmarketing/image/filter/ImageFilterApiImpl.java

```diff
@@ -113,6 +131,31 @@ public Dimension getWidthHeight(final File imageFile) {
 
     }
 
+    @VisibleForTesting
+    Dimension getWidthHeightofSVG(@Nonnull final File imageFile) {
+        SAXSVGDocumentFactory factory = new SAXSVGDocumentFactory(
+                XMLResourceDescriptor.getXMLParserClassName());
+
+        try (InputStream is = Files.newInputStream(imageFile.toPath())) {
+            Document document = factory.createDocument(imageFile.toURI().toString(), is);
+
+            String viewBox = document.getDocumentElement().getAttribute("viewBox");
+            String[] viewBoxValues = viewBox.split(" ");
+
+            float x = Float.parseFloat(viewBoxValues[2]);
```

Comment:
here should check if the length is greater than 3 to avoid out of range exception

Reply:
It was intended to get caught by the Exception and throw an error.